contents/docs/data/events.mdx

```diff
@@ -25,6 +25,20 @@ You can view the properties of each event by clicking on the items in the ‘Eve
 
 You can also click each user to view a their entire event history.
 
+## Default events and properties
+
+You may notice that some events or properties are prefixed with `$` e.g., `$pageview`, `$feature_flag_called`, `$current_url`, or `$os_version`. This indicates that these are default PostHog events or properties.
+
+### Pageview events
+
+By default, pageview events are automatically sent when using the [JavaScript snippet or SDK](/docs/libraries/js). You can disable this by setting `capture_pageview: false` in the [config](/docs/libraries/js#config).
+
+You can manually send pageview events by calling `posthog.capture('$pageview')`. You'll need to do this in single-page applications like [React](/docs/libraries/react), [Next.js](/docs/libraries/next-js#pages-router) or [Vue](/docs/libraries/vue-js#capturing-page-views), since otherwise a pageview event will only be sent for the first page.
```

Comment:
```suggestion
You can manually send pageview events by calling `posthog.capture('$pageview')`. This is necessary for single-page applications like [React](/docs/libraries/react), [Next.js](/docs/libraries/next-js#pages-router) or [Vue](/docs/libraries/vue-js#capturing-page-views), since otherwise a pageview event is only sent for the first page.
```